### PR TITLE
docs: add page on grafana/influxdb dashboard

### PR DESCRIPTION
**Pull request by jmcook1186**
Adds a Grafana dashboard page to the docs.

### CODE REVIEW

File: docs/_interface/dashboards.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+the data.
+
+## Setting up InfluxDB {#setting-up-influxdb}
+
+InfluxDB can be downloaded from the [Influxdata release page](https://portal.influxdata.com/downloads/). 
+It can also be installed from a [repository](https://repos.influxdata.com/). 
+For example for a Debian based Linux operating system:
+
+```sh
+curl -tlsv1.3 --proto =https -sL https://repos.influxdata.com/influxdb.key | sudo apt-key add
```
Comment (holiman) [inline docs/_interface/dashboards.md:35]: Very explicit instructions like these easily bitrot, might be better to link to the official download/install docs for influx?

File: docs/_interface/dashboards.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+With the InfluxDB database setup and successfully receiving data from Geth, 
+the next step is to install Grafana so that the data can be visualized. 
+Instructions for specific operating systems are available on the Grafana 
+[downloads page](https://grafana.com/grafana/download?pg=get&plcmt=selfmanaged-box1-cta1).
+
+Alternatively, the following code snippet shows how to download, install and 
+run Grafana on a Debian based Linux system:
+
+```sh
```
Comment (holiman) [inline docs/_interface/dashboards.md:116]: .. And same here.. ?

**Review by holiman — APPROVED** [review]
LGTM, two suggestions but feel free to do as you want with those

File: docs/_interface/dashboards.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+exit
+```
+
+InfluxDB is running and configured to store metrics from Geth.
+
+## Preparing Geth {#preparing-geth}
+
+After setting up database, metrics need to be enabled in Geth. 
+Various options are available, as documented in the `METRICS AND STATS OPTIONS` 
+in `geth --help` and in our [metrics page](). In this case Geth will be configured
```
Comment (jmcook1186) [inline docs/_interface/dashboards.md:90]:
```suggestion
in `geth --help` and in our [metrics page](/docs/interface/metrics). In this case Geth will be configured 
```

**Pull request merged**